content/ADR-175-remove-assets-from-build-local-addressables.md

```diff
@@ -0,0 +1,128 @@
+  - on Standalone "Streaming Assets" is a directory alongside a main binary, and, thus, are loaded by reading from disk
+
+### Retrocompatibility and Versioning
+Currently the renderer is a self-contained experience; and as for "Local" Addressables no dedicated versioning is required:
+  - on WebGL "Streaming Assets" are a part of the npm package
+  - on Standalone "Streaming Assets" are distributed within the same archive
+
+### Addressables loading
+Depending on the specific case, we will use all possible ways to load assets:
+  - [Label](https://docs.unity3d.com/Packages/com.unity.addressables@1.21/manual/Labels.html) (to load a list of similar assets)
+  - Address (to load an asset by the hardcoded string)
+     - By default, the address is a path to the file
+<figure>
+  <img src="/resources/ADR-175/AddressableAddress.png" />
+</figure>
+
+     - It is desirable to shorten the address and adjust it according to the directories and groups structure. The exact rules of the former will be elaborated on in the production pipeline
+  - GUID (to load an asset by "Asset Reference")
+
+### [Groups configuration](https://docs.unity3d.com/Packages/com.unity.addressables@1.21/manual/GroupSettings.html)
+Group settings determine how the assets in a group are treated in content builds.
+
+<figure>
+  <img src="/resources/ADR-175/GroupsConfiguration.png" />
+</figure>
+
+We are going to distribute assets in "Addressable Asset Groups" according to the following rules:
+   - "Essentials" corresponds to the assets that are mandatory for the application to function (e.g. Skybox) and can't be excluded
+   - Other assets according to the logical designation (e.g. "Fonts") and usage (e.g. "Window XXX Resources") fall into different groups; without them the application can function
+   - Assets packed together/separately according to the required loading way: any combination of "Include Addresses in Catalog", "Include GUIDs in Catalog" and "Include Labels in Catalog". Thus, we will reduce the size of Catalog.
+   - Bundle Mode "Pack Together by Label" will be used to differentiate between platforms (WebGL, Standalone) within the same logical group to exclude unused assets (and the related overhead) if needed
+   - We may come up with additional rules for our convenience
+
+Check [Unity guidelines](https://docs.unity3d.com/Packages/com.unity.addressables@1.21/manual/AddressableAssetsDevelopmentCycle.html#organizing-addressable-assets) for additional information.
+
+#### Asset Bundle CRC	
+At the moment we don't have any sensitive data that requires integrity check. Moreover, for bigger files it will cause noticeable overhead on loading.
```

Comment:
```suggestion
At the moment, we don't have any sensitive data that requires an integrity check. Moreover, for bigger files, it will cause noticeable overhead on loading.
```